### PR TITLE
[AAE-14850] Fix outcome buttons position

### PR DESCRIPTION
**Pull request by DudaRobert**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/AAE-14850

**What is the new behaviour?**

With the proposed change, this is how the start event looks like with and without form:

Screenshot 2023-05-30 at 16 26 29
Screenshot 2023-05-30 at 17 08 46

Here's also start event with form with custom outcomes:
Screenshot 2023-05-30 at 17 36 47

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.scss [hunk 1]
```diff
@@ -4,13 +4,9 @@
             color: var(--adf-theme-mat-grey-color-a200-dark);
         }
 
-        mat-card-actions {
+        &-cloud-actions {
             display: flex;
-            justify-content: flex-end;
-
-            .mat-button {
```
Comment (DudaRobert) [inline lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.scss:11]: noticed that this wasn't affecting anything

Comment (DudaRobert) [reply lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.scss:11]: Don't know if I understand what you're asking about, but what I meant with my comment is that I've tested the styles that I deleted and noticed that their presence doesn't change anything in the elements they're describing.

Comment (DudaRobert) [reply lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.scss:11]: You're right, I almost reverted you previous fix

File: lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.scss [hunk 2]
```diff
@@ -5,13 +5,19 @@
         }
 
         mat-card-actions {
+            text-align: right;
```
[the remark this answers is not in the document]
Comment (DudaRobert) [reply lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.scss]: Must've added by mistake. Fixed now.

**Review by BSekula — APPROVED** [review]

**Pull request merged**